admin_ui_support/modules/jsonapi_support/src/JsonapiSupportServiceProvider.php

```diff
@@ -23,12 +22,6 @@ public function alter(ContainerBuilder $container) {
       $definition = $container->getDefinition('jsonapi.resource_type.repository');
       $definition->setClass(ResourceTypeRepository::class);
     }
-
-    if ($container->has('jsonapi.request_handler')) {
-      // Override the class used for Jsonapi request handler.
-      $definition = $container->getDefinition('jsonapi.request_handler');
-      $definition->setClass(JsonApiRequestHandler::class);
```

Comment:
Our RequestHandler class was not actually being used with the change to JSON API 2.x. 

That means that our 2 classes that extended `EntityResource` were not actually being invoked. For `ConfigEntityResource.php` this doesn't affect the app because we are actually using this functionality.
It may take some work to get this class to work again with 2.x. 

I say we remove it here and add it back later if/when we are actually going to use it.

For `CrossBundleEntityResource` we only needed this because we were overriding `getCollectionQuery`. We no longer need to do this.